inflection/src/inflection/dialog/InflectableStringConcept.cpp

```diff
@@ -15,7 +15,21 @@
 
 namespace inflection::dialog {
 
-InflectableStringConcept::InflectableStringConcept(const SemanticFeatureModel* model, const SpeakableString& value)
+InflectableStringConcept::InflectableStringConcept(
+    const SemanticFeatureModel* model,
+    const SpeakableString& value,
+    const ::std::map<SemanticFeature, ::std::u16string> intitialConstraints
+)
+    : super(model)
+    , value(value)
+    , defaultDisplayValue(value, *npc(super::getSpeakFeature()), intitialConstraints)
+{
+}
+
+InflectableStringConcept::InflectableStringConcept(
+    const SemanticFeatureModel* model,
+    const SpeakableString& value
+)
```

Comment:
Can you please declare these functions in the same order as the header? It makes it easier to navigate.

Also the 2 argument constructor with the SpeakableString can now call the new constructor with an empty map. Then the copied code can be removed. It's best to keep the copy and paste code to a minimum to improve the maintenance.